src/models/stackedAreaChart.js

```diff
@@ -375,11 +375,15 @@ nv.models.stackedAreaChart = function() {
                 //If we are in 'expand' mode, force the format to be a percentage.
                 var valueFormatter = (stacked.style() == 'expand') ?
                     function(d,i) {return d3.format(".1%")(d);} :
-                    function(d,i) {return yAxis.tickFormat()(d); };
+                    tooltip.valueFormatter(); 
+                
+                var headerFormatter = tooltip.headerFormatter();
+
                 interactiveLayer.tooltip
                     .position({left: pointXLocation + margin.left, top: e.mouseY + margin.top})
                     .chartContainer(that.parentNode)
                     .valueFormatter(valueFormatter)
+                    .headerFormatter(headerFormatter)
```

Comment:
Again, it shoudn't be using chart.tooltip for the value...  we should just give a value as a default here by itself.   

One thing is that if you do chart.update() it'll probably overwrite these values... if it does, then use the _headerFormater() and _valueFormatter() versions of the options... adding an underscore to any option makes it only save the value if the normal option (without the underscore) has never been called.